mono/profiler/mono-profiler-logging.c

```diff
@@ -5633,7 +5644,10 @@ setup_user_options (const char *arguments) {
 	if (profiler->action_flags.track_stack) {
 		profiler->flags |= MONO_PROFILE_ENTER_LEAVE;
 	}
-	
+    if (profiler->action_flags.track_fileio) {
+        profiler->flags |= MONO_PROFILE_FILEIO;
+    }
```

Comment:
Nit: Is there a tabs vs. spaces issue here? I've had problems in Mono code when my editor is setup up for spaces in Unity.